src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 403px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
```

Comment:
white is default color for background

Reply:
I changed that, look at the second commit made yesterday, please.